komodo/check_up_to_date_pypi.py

```diff
@@ -281,6 +292,15 @@ def get_args() -> argparse.Namespace:
             "If given, will ignore packages with names that match the regex expression"
         ),
     )
+    parser.add_argument(
+        "--target-platform",
+        default=sys.platform,
+        choices=["darwin", "win32", "linux"],
+        help=(
+            "Which OS the new package version should be compatible with. Should be either linux,"
+            " macos, or win32. Defaults to OS of host machine"
```

Comment:
small thing - the choices do not match the help

Reply:
Fixed!